acquia/blt
# Travis memcache

Fixes #3077 
--------

Changes proposed:
---------
Start the memcache service on Travis and add the memcached.so php extension to php.ini to allow for using memcache in Travic. 
-
- Note: In looking over the travis config, it doesnt look like apc is installed or configured which may impact this work. I will open a separate issue if it looks like there is a bug. 

Steps to test
----------
1. Start travis build
2. Observe "Added memcached php extension" in log.
